.rubocop.yml

```diff
@@ -1,3 +1,6 @@
+FrozenStringLiteralComment:
```

Comment:
Made the massive PR here: https://github.com/zendesk/samson/pull/1200

@zendesk/samson lemme know what you want to do...